spec/std/file_spec.cr

```diff
@@ -967,7 +979,7 @@ describe "File" do
       File.writable?("foo\0bar")
     end
 
-    it_raises_on_null_byte "executable?" do
+    it_raises_on_null_byte "executable?", pending_win32: true do
```

Comment:
Just duplicating the content of `it_raises_on_null_byte` wrapped in `pending_win32 do ... end` would be much simpler code than adding an extra argument for a single use.